README.md

```diff
@@ -1,39 +1,89 @@
-# Role Name
-
-A brief description of the role goes here.
+# zfs-snap-manager
 
+This role installs and configures [zfs-snap-manager](https://github.com/khenderick/zfs-snap-manager), a tool to automatically create, replicate and destroy ZFS snapshots.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
-
+`python2` has to be installed on the target machine. Obviously, any ZFS filesystems that should be snapshotted must exist.
+The machine that executes the playbook needs to have internet access as well as `git` installed.
```

Comment:
Why don't you just install git? "The control machine needs to have internet access and needs to have git installed if it's not a Debian-based distribution"